Packs/Nexthink/Integrations/Nexthink/Nexthink.yml

```diff
@@ -0,0 +1,121 @@
+category: Utilities
+commonfields:
+  id: Nexthink
+  version: -1
+configuration:
+- defaultvalue: connector.nexthink.com
+  display: Engine Host (e.g. connector.nexthink.com)
+  name: url
+  required: true
+  type: 0
+- defaultvalue: "1671"
+  display: Nexthink Connector Port
+  name: port
+  required: false
+  type: 0
+- display: Username
+  name: credentials
+  required: true
+  type: 9
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: Nexthink helps IT teams deliver on the promise of the modern digital
+  workplace. Nexthink is the only solution to provide enterprises with a way to visualize,
+  act and engage across the entire IT ecosystem to lower IT cost and improve digital
+  employee experience.
+display: Nexthink
+name: Nexthink
+script:
+  commands:
+  - name: nt_compliance_check
+    arguments:
+    - name: ipaddress
+      required: true
+      description: Endpoint IP Address
+      defaultValue: None
+    - name: hostname
+      required: true
+      default: true
+      description: Endpoint Hostname
+      defaultValue: None
+    outputs:
+    - contextPath: Nexthink.Compliance.Device Name
+      description: Endpoint Device Host Name
+    - contextPath: Nexthink.Compliance.Last Logged On User
+      description: Last Logged On User
+    - contextPath: Nexthink.Compliance.IP Address
+      description: Endpoint IP Address
+    - contextPath: Nexthink.Compliance.MAC Address
+      description: Endpoint MAC Address
+    - contextPath: Nexthink.Compliance.Device Antivirus
+      description: Endpoint Antivirus Name
+    - contextPath: Nexthink.Compliance.Device Antivirus RTP
+      description: Endpoint Antivirus Real Time Protection Status
+    - contextPath: Nexthink.Compliance.Device Antivirus Updated
+      description: Endpoint Antivirus Update Status
+    - contextPath: Nexthink.Compliance.Device Antispyware
+      description: Endpoint Antispyware Name
+    - contextPath: Nexthink.Compliance.Device Antispyware RTP
+      description: Endpoint Antispyware Real Time Protection Status
+    - contextPath: Nexthink.Compliance.Device Antispyware Updated
+      description: Endpoint Antispyware Update Status
```

Comment:
ContextPath cannot include whitespaces. For example,
Nexthink.Compliance.Device Name should be changed to Nexthink.Compliance.DeviceName.
Please change all outputs in this file to match this convention. 
In addition, for each output you should add the `type` field of the outputs. Type can be string/number/date/boolean.
for example: 
    - contextPath: Nexthink.Compliance.DeviceName
      description: Endpoint Device Host Name
      type: string
Please change here and in all places in this file. You can see more examples in every integration that we have in Content.

Reply:
done